apps/studio/styles/monaco.css

```
@@ -25,6 +25,7 @@
 .monaco-diff-editor {
   --vscode-editor-background: hsl(var(--background-dash-sidebar)) !important;
   --vscode-editorGutter-background: hsl(var(--background-dash-sidebar)) !important;
+  outline: none !important;
```

Stylelint (17.13.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)